azure/biganimal-preflight-azure

```diff
@@ -42,6 +42,7 @@
 #  https://www.enterprisedb.com/docs/biganimal/latest/getting_started/01_check_resource_limits/#increasing-network-quota
```

Comment:
```suggestion
#  https://www.enterprisedb.com/docs/biganimal/latest/getting_started/01_check_resource_limits
```
There is no longer a section titled "Increasing network quota" in the "Check resource limits" topic.